docs/src/security/gdpr.md

```diff
@@ -9,18 +9,39 @@ description: |
-* **Privacy Impact Assessment (PIA)**: We have performed an internal PIA to ensure that we comply with GDPR principles and obligations.
-* **Security**: We have created [https://platform.sh/security](https://platform.sh/security) to document our security features.
-* **Data Collection**: We have documented information about [what data we collect](/security/data-collection.md).
-* **Data Retention**: We have documented information about our [data retention](/security/data-retention.md) practices.
-* **Data Processing Agreement (DPA)**: We have revised our [Terms of Service](https://platform.sh/tos) and [Privacy Policy](https://platform.sh/privacy-policy) to align with the GDPR, and we offer a pre-signed DPA agreement that can be downloaded at the top of the [Privacy Policy](https://platform.sh/privacy-policy).
+* **Personally identifiable information (PII)**: We audited our systems to confirm that your personal data is encrypted and protected. 
+* **Data Flows**: We identified and classified data, and created a high-level data flow diagram that maps out data shared with vendors, including cross-border transfers.
+* **Privacy Impact Assessment (PIA)**: We perform internal PIAs to ensure that we comply with GDPR principles and obligations.
+* **Security**: We created [https://platform.sh/security](https://platform.sh/security) to document our security features.
+* **Data Collection**: We documented information about [what data we collect](/security/data-collection.md).
+* **Data Retention**: We documented information about our [data retention](/security/data-retention.md) practices.
+* **Data Processing Agreement (DPA)**: We revised our [Terms of Service](https://platform.sh/tos) and [Privacy Policy](https://platform.sh/privacy-policy) to align with the GDPR, and we offer a pre-signed DPA agreement that can be downloaded at the top of the [Privacy Policy](https://platform.sh/privacy-policy).
+* **Audits:** We undergo a yearly third-party audit of our privacy practices which includes GDPR.
+
+
+
+# Controller vs Processor
+
+With respect to the EU GDPR, Platform.sh is both a Controller and Processor:
+
+* We are a Controller for the overall service and in particular when we are in charge of data subjects who are explicitly the users of our services.
+* We are a Processor for our customers (Controllers) that are in charge of data subjects that they have collected.
+
+In technical terms, we operate a:
+
+* Infrastructure Control Plane - This is Platform.sh’s orchestration, control, and management environment.
+* Customer Data Plane - This the customer’s data app and data environment.
+
+We are the Controller for the Infrastructure Control Plane (and our PaaS service). We are the Processor for the Customer Data Plane. 
+
+GDPR Data Processing Agreements (DPAs) and EU Standard Contractual Clauses executed with our customers apply to our processing activities contained in the Customer Data Plane.
```

Comment:
Since we use `Term (Acromym)` everywhere else, I'd add `(SCCs)` here too for consistency.

```suggestion
GDPR Data Processing Agreements (DPAs) and EU Standard Contractual Clauses (SCCs) executed with our customers apply to our processing activities contained in the Customer Data Plane.
```